Classes/Console/Service/Configuration/ConfigurationService.php

```diff
@@ -162,12 +162,17 @@ public function removeLocal($path)
      * @param string $path
      * @param mixed $value
      * @param string $targetType
+     * @param bool $asJson Treat value as JSON and overwrites $targetType
      * @return bool
      */
-    public function setLocal($path, $value, $targetType = '')
+    public function setLocal($path, $value, $targetType = '', $asJson = false)
     {
         try {
-            $value = $this->convertToTargetType($path, $value, $targetType);
+            if ($asJson) {
```

Comment:
ConfigurationService can stay untouched and this if should be moved to ConfigurationCommandController